boards/arm/frdm_k32l2b3/doc/index.rst

```diff
@@ -0,0 +1,230 @@
+.. _frdm_k32l2b3:
+
+NXP FRDM-K32L2B3
+##############
+
+Overview
+********
+
+The FRDM-K32L2B3 Freedom development board provides a platform for evaluation
+and development of the K32 L2B MCU Family. The board includes onboard debug
+probe, segment LCD, accelerometer/magnetometer, a full speed USB and easy
+access to K32 L2B's MCU I/O. The FRDM-K32L2B3 board is fully supported by the
+MCUXpresso suite of tools, which provides device drivers, middleware and
+examples to allow rapid development, plus configuration tools and an optional
+free IDE.
+
+.. image:: ./frdm_k32l2b3.jpg
```

Comment:
the filename uploaded has a dash in it: ``frdm-k32l2b3.jpg`` so rename the image filename or fix this name to match

Reply:
Thank you, @dbkinder . just updated the doc accordingly.